dbms/sqlquery/mysql/src/cloud/benchflow/monitors/querymysql.go

```diff
@@ -35,35 +36,40 @@ func queryHandler(w http.ResponseWriter, r *http.Request) {
     }
     // Checks over the rows if the comparison holds
     rowI := 0
-    response := Response{false}
+    result := false
+    queryResponse := ""
     for rows.Next() {
```

Comment:
Can't you do some assumption on the structure of the response, and directly pick the value for comparison? This way you are comparing line by line and you get it correct only if the actual value you want to compare is on the last line of the result.